docs/products/valkey/howto/connect-valkey-cli.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Connect with valkey-cli
+---
+
+Learn how to establish a connection to an Aiven for Valkey™ service using the `valkey-cli`.
+
+
+## Variables
+
+Replace the following placeholders in the code sample with actual values
+from your service overview page:
+
+| Variable    | Description                                              |
+| ----------- | -------------------------------------------------------- |
+| `SERVICE_URI` | URI for the Aiven for Valkey service connection  |
+
+## Prerequisites
+
+For this example, you will need:
+
+1.  The `valkey-cli` client installed. You can install this as part of
+    [Valkey server installation](https://valkey.io/topics/installation/).
+
+## Code
+
+Execute the following command from a terminal window to connect:
```

Comment:
Google's style guide recommends using "run" instead of "execute" for simplicity.